vault/expiration.go

```diff
@@ -1859,6 +1879,11 @@ func (m *ExpirationManager) loadEntryInternal(ctx context.Context, leaseID strin
 	}
 	le.namespace = ns
 
+	if le.isZombie() {
+		m.zombies.Store(le.LeaseID, le)
+		return le, nil
+	}
```

Comment:
I wonder if we can push this into the updatePending function, which will allow us to have one place where we can update the pending, nonexpiring, and zombie maps.

Reply:
https://hashicorp.atlassian.net/browse/VAULT-2303